src/landsat_importer/main.py

```diff
@@ -139,7 +139,9 @@ def main():
 
         if slurm_options:
             script_contents = "conda activate rioxarray_env\n\n"
-            script_contents += f"run_landsat_importer {input_path} {output_path} --bands {args.bands}"
+            script_contents += f"run_landsat_importer {input_path} {output_path} "
+            if args.bands:
+                script_contents == "--bands {' '.join(args.bands)}"
```

Comment:
The slurm stuff should probably be moved to a separate helper / submission script and not exist inside the main landsat_importer code. (but that's a separate problem for later!)

Reply:
Agreed - I have removed the slurm stuff from this program!